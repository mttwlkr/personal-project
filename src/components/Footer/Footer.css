.footer {
  margin-top: 15px;
  background: #efefef;
}

p {
  margin: 0;
}
