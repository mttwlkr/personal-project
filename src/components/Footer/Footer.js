import React from 'react';
import './Footer.css';
import Facebook from '../../assets/facebook.svg';
import Instagram from '../../assets/instagram.svg';
import YouTube from '../../assets/youtube.svg';

const Footer = () => {

  return (
    <footer className='footer'>
      <p>&copy; 2018 Slvsh LLC | All rights reserved</p>
      <section className='social-media-div'>
        
        <a target='_blank' 
          href='https://www.instagram.com/theslvsh'
          rel='noopener noreferrer'>
          <img 
            className='social-media-logos sml-ig' 
            src={Instagram}
            alt='instagram'
          />
        </a>

        <a target='_blank'
          href='https://youtube.com/theslvsh'
          rel='noopener noreferrer'>
          <img 
            className='social-media-logos sml-yt' 
            src={YouTube}
            alt='you-tube'
          />
        </a>

        <a target='_blank' 
          href='https://www.facebook.com/theslvsh'
          rel='noopener noreferrer'>
          <img 
            className='social-media-logos sml-fb' 
            src={Facebook}
            alt='facebook'
          />
        </a>
      </section>
    </footer>
  )
}

export default Footer

