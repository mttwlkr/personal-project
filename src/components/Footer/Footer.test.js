import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer.js';

describe('footer', () => {

  let wrapper;

  it.skip('should match the snapshot', () => {
    wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})

