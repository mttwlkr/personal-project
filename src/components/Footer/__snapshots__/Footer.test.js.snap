// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          © 2018 Slvsh LLC | All rights reserved
</p>,
        <section>
          <p>
                    Follow SLVSH
          </p>
          <a
                    href="https://www.facebook.com/theslvsh"
                    rel="noopener noreferrer"
                    target="_blank"
          >
                    Facebook
          </a>
          <a
                    href="https://www.instagram.com/theslvsh"
                    rel="noopener noreferrer"
                    target="_blank"
          >
                    Instagram
          </a>
          <a
                    href="https://twitter.com/theslvsh"
                    rel="noopener noreferrer"
                    target="_blank"
          >
                    Twitter
          </a>
</section>,
      ],
      "className": "footer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "© 2018 Slvsh LLC | All rights reserved",
        },
        "ref": null,
        "rendered": "© 2018 Slvsh LLC | All rights reserved",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              Follow SLVSH
</p>,
            <a
              href="https://www.facebook.com/theslvsh"
              rel="noopener noreferrer"
              target="_blank"
>
              Facebook
</a>,
            <a
              href="https://www.instagram.com/theslvsh"
              rel="noopener noreferrer"
              target="_blank"
>
              Instagram
</a>,
            <a
              href="https://twitter.com/theslvsh"
              rel="noopener noreferrer"
              target="_blank"
>
              Twitter
</a>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Follow SLVSH",
            },
            "ref": null,
            "rendered": "Follow SLVSH",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Facebook",
              "href": "https://www.facebook.com/theslvsh",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "Facebook",
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Instagram",
              "href": "https://www.instagram.com/theslvsh",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "Instagram",
            "type": "a",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Twitter",
              "href": "https://twitter.com/theslvsh",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": "Twitter",
            "type": "a",
          },
        ],
        "type": "section",
      },
    ],
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            © 2018 Slvsh LLC | All rights reserved
</p>,
          <section>
            <p>
                        Follow SLVSH
            </p>
            <a
                        href="https://www.facebook.com/theslvsh"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        Facebook
            </a>
            <a
                        href="https://www.instagram.com/theslvsh"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        Instagram
            </a>
            <a
                        href="https://twitter.com/theslvsh"
                        rel="noopener noreferrer"
                        target="_blank"
            >
                        Twitter
            </a>
</section>,
        ],
        "className": "footer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "© 2018 Slvsh LLC | All rights reserved",
          },
          "ref": null,
          "rendered": "© 2018 Slvsh LLC | All rights reserved",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                Follow SLVSH
</p>,
              <a
                href="https://www.facebook.com/theslvsh"
                rel="noopener noreferrer"
                target="_blank"
>
                Facebook
</a>,
              <a
                href="https://www.instagram.com/theslvsh"
                rel="noopener noreferrer"
                target="_blank"
>
                Instagram
</a>,
              <a
                href="https://twitter.com/theslvsh"
                rel="noopener noreferrer"
                target="_blank"
>
                Twitter
</a>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Follow SLVSH",
              },
              "ref": null,
              "rendered": "Follow SLVSH",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Facebook",
                "href": "https://www.facebook.com/theslvsh",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "Facebook",
              "type": "a",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Instagram",
                "href": "https://www.instagram.com/theslvsh",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "Instagram",
              "type": "a",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Twitter",
                "href": "https://twitter.com/theslvsh",
                "rel": "noopener noreferrer",
                "target": "_blank",
              },
              "ref": null,
              "rendered": "Twitter",
              "type": "a",
            },
          ],
          "type": "section",
        },
      ],
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
